websites/V/Vrc.tl/presence.ts

```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+		clientId: "1201575439974543410",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	regexes = {
+		detail: /\/admin\/event\/detail\/(\d+)/,
+		index: /\/([0-9]+(\|[0-9]+)+)*/,
+	};
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://files.catbox.moe/dalqh3.png",
+		smallImageKey: "https://files.catbox.moe/g66t9n.png",
```

Comment:
This asset has a standardized version available
```suggestion
		smallImageKey: Assets.Search,
```